### DISCUSSION TITLE
Not being able to submit project submission links like github repository url , project demo url , project video url

### DISCUSSION THREAD

**Discussion by megh212** [github/india#1074] [Students]
I am not being able to update these links 

![2022-12-25 09_32_30-Learning _ Future Ready Talent](https://user-images.githubusercontent.com/88397453/209456630-f43f6d19-ac78-4809-9bd4-96ce2972630e.png)
 
My project evaluation status is projected not submitted 
as the deadline for project submission is approaching please help me

**Comment by FRTSupport** [comment]
Our backend team has resolved the issue of project submission by bringing the placeholders to full functionality.

**Comment by Sroyals** [comment]
Hello I faced kind off similar problem but my project submission is not active
I am not able to put github repositary url, and all plz help
